boards/arduino-uno/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+#![no_std]
+
+pub extern crate atmega328p_hal as hal;
+
+mod pins;
+
+pub use atmega328p_hal::atmega328p;
+pub use crate::atmega328p::Peripherals;
+pub use atmega328p_hal::prelude;
+pub use crate::pins::*;
+
+pub type Delay = hal::delay::Delay<hal::clock::MHz16>;
+pub type Serial<IMODE> = hal::usart::Usart0<hal::clock::MHz16, IMODE>;
+pub type I2c<M> = hal::i2c::I2c<hal::clock::MHz16, M>;
```

Comment:
Just in case:  Uno is using a 16MHz clock by default as well? (I don't know, just want to make sure)

Reply:
Just confirmed on the official product page that it does use a 16MHz quartz crystal.